Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM registry.access.redhat.com/ubi8/python-38 AS demo
+
+COPY ./ /trestle-bot
+
+WORKDIR /trestle-bot
+
+USER root
+
+# Install dependencies
+RUN  python3.8 -m pip install --upgrade pipx \
+     && pipx install poetry==1.5.1 \
+     && poetry config virtualenvs.create false \
+     && poetry install --without tests,dev
+
+RUN  chown -HR 1001:1001 /trestle-bot \
+     chown -HR 1001:1001 /opt/app-root/src/
+     
+USER 1001
+
+ENTRYPOINT poetry run trestle-bot \
```

Comment:
Can you update to valid JSON here? https://docs.kics.io/latest/queries/dockerfile-queries/b86987e1-6397-4619-81d5-8807f2387c79/

Reply:
Resolving. We need to use shell form because we need environmental substitution.